### PR TITLE
chore(scheduler-docs): add the Views Guide

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-react-scheduler/docs/guides/view-types.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The following plugins implement view features:
+
+- [DayView](../reference/day-view.md) - renders a day layout
+- [WeekView](../reference/week-view.md) - renders a week layout
+- [MonthView](../reference/month-view.md) - renders a month layout
+
+## DayView
+
+The Day View displays appointments for a specified date. The Scheduler component arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other. To implement this view add the `DayView` plugin.
```
[the remark this answers is not in the document]
Comment (SergeyAlexeev) [reply packages/dx-react-scheduler/docs/guides/view-types.md]: > The Day View displays appointments...

The Day view displays appointments... If you are talking about an abstract day view, not about a plugin.
The DayView plugin displays appointments... If you are talking about a plugin.

File: packages/dx-react-scheduler/docs/guides/view-types.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+By default the length of time scale is 24 hours. To change this value define the view plugin's properties `startDayHour` and `endDayHour`. In the following example the start of time scale is 9 a.m. and end of it is 7 p.m..
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/guides/view-types.md]: > By default the length...
By default the duration

File: packages/dx-react-scheduler/docs/guides/view-types.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+.embedded-demo({ "path": "scheduler-basic/day", "showThemeSelector": true })
+
+## Week View
+
+The Week View display appointments for a week. The appointments layouting is same with the Day View feature.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/guides/view-types.md]: > The appointments layouting is same with the Day View feature.

On this view, appointments are rendered like on the day view.

File: packages/dx-react-scheduler/docs/guides/view-types.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+.embedded-demo({ "path": "scheduler-basic/week", "showThemeSelector": true })
+
+The work week view is a most common request. To implement this case define the `excludedDays` property of the [WeekView](../reference/week-view.md) plugin with `[0, 6]`.
+
+## MonthView
+
+The Month View displays appointments for a month. The Scheduler component arranges appointments from left to right. An appointment's size depends on its duration in days. However, it occupies the entire day cell if an appointment lasts only for several hours or minutes. The time scale and all-day panel are not displayed in this view.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/guides/view-types.md]: > The Month View displays appointments
The same about naming.

File: packages/dx-react-scheduler/docs/guides/view-types.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# React Scheduler - View Types
+
+The Scheduler component provides several types of views. Views display information in various ways, you can choose the most convenient view for your data source or combine few view and switching between them. This topic describes visual elements and their position for each view.
```
[the remark this answers is not in the document]
Comment (ushkal) [reply packages/dx-react-scheduler/docs/guides/view-types.md]: We can add here about what's the difference between views - time intervals length and appointments layout

File: packages/dx-react-scheduler/docs/guides/view-types.md (hunk 4 above)
Comment (SergeyAlexeev) [reply packages/dx-react-scheduler/docs/guides/view-types.md]: Common question: is it OK if we copy text from js.devexpress.com? ))

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/view-types.md]: Why not? It's an easy way. After review by Roman and Dirk, these texts will not similar.

Comment (SergeyAlexeev) [reply packages/dx-react-scheduler/docs/guides/view-types.md]: > After review by Roman and Dirk, these texts will not similar

Are you sure?)

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+- [DayView](../reference/day-view.md) - renders a day layout
+- [WeekView](../reference/week-view.md) - renders a week layout
+- [MonthView](../reference/month-view.md) - renders a month layout
+
+## DayView
+
+The `DayView` displays appointments for a specified date. The Scheduler component arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other.
+
+By default the duration of time scale is 24 hours. To change this value define the view plugin's properties `startDayHour` and `endDayHour`. In the following example the start of time scale is 9 a.m. and end of it is 7 p.m..
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/guides/views.md]: > the start of time scale is 9 a.m. and end of it is 7 p.m..

a day starts at 9 a.m. and ends at 7 p.m.

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+The Scheduler component provides several types of views. Views display information in various ways, you can choose the most convenient view for your data source or combine few views and switching between them.
+
+## Related Plugins
+
+The following plugins render corresponding views:
+
+- [DayView](../reference/day-view.md) - renders a day layout
+- [WeekView](../reference/week-view.md) - renders a week layout
+- [MonthView](../reference/month-view.md) - renders a month layout
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/guides/views.md]: I'm not sure, but what about the `ViewState` one? It affects all views. Its name starts with `View`, at least))

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/views.md]: This plugin is not required for basic setup. I think we should not add it.

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+# React Scheduler - Views
+
+The Scheduler component provides several types of views. Views display information in various ways, you can choose the most convenient view for your data source or combine few views and switching between them.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/views.md]: switching -> switch

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+The following plugins render corresponding views:
+
+- [DayView](../reference/day-view.md) - renders a day layout
+- [WeekView](../reference/week-view.md) - renders a week layout
+- [MonthView](../reference/month-view.md) - renders a month layout
+
+## DayView
+
+The `DayView` displays appointments for a specified date. The Scheduler component arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/views.md]: We can add here that a Day view is the most detailed. And the opposite in Month view section

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+By default the duration of time scale is 24 hours. To change this value define the view plugin's properties `startDayHour` and `endDayHour`. In the following example a time scale starts at 9 a.m. and ends at 7 p.m.
```
Comment (ushkal) [inline packages/dx-react-scheduler/docs/guides/views.md]: I think we should define a concept of a time scale somewhere - it's common for all views

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/views.md]: The time scale describes on the image)

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/views.md]: I can insert a reference to Fundamentals - Overview. The information about the time scale will be there.

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
+# React Scheduler - Views
+
+Views arrange appointments in different layouts. The Scheduler provides several view types. You can choose a single view type that presents your data in the best way or allow users to switch between several view types.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/views.md]: "that presents your data in the best way" This is vague

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+## Related Plugins
+
+The following plugins render particular views:
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/views.md]: "render particular views" -> render views

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+- [DayView](../reference/day-view.md) - renders a day view
+- [WeekView](../reference/week-view.md) - renders a week view
+- [MonthView](../reference/month-view.md) - renders a month view
+
+## Day View
+
+The `DayView` displays appointments for a specific date.
+
+The Scheduler arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/views.md]: "their width is decreased and they are placed next to each other" -> their widths decrease and they are placed next to each other

File: packages/dx-react-scheduler/docs/guides/views.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+
+.embedded-demo({ "path": "scheduler-basic/week", "showThemeSelector": true })
+
+You can use the [WeekView](../reference/week-view.md) plugin's `excludedDays` property to exclude specific days from the week view. For example, to implement a work week view, set this property to `[0, 6]`.
+
+## Month View
+
+The `MonthView` displays appointments for a month.
+
+The Scheduler arranges appointments left to right. An appointment's size depends on its duration in days. Even if an appointment lasts less than a day, it occupies the entire day cell.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/guides/views.md]: "Even if an appointment lasts less than a day, it occupies the entire day cell." -> Appointments that last less than a day occupy the entire day cell.

**Pull request merged**